packages/vue-test-workspace-vue-2/vue-tsc/strict-template/unknownProp/main.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+	<div>
+		<!-- @vue-expect-error -->
+		<Foo bar="123"></Foo>
+
+		<!-- @vue-expect-error -->
+		<div foo="123"></div>
+	</div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+const Foo = defineComponent({
+	props: {
+		foo: String
+	}
+});
+</script>
+<script setup lang="ts">
+</script>
```

Comment:
Note that this is a bit different from how I wrote this in my changes. My version only had one `script setup` block.